todos/week1/2_variables_and_typestodo.py

```diff
@@ -35,10 +40,9 @@
 # The type() function is an example of a polymorphic function, meaning that the same
 # function name can be used for different types. BIG. WORDS. GET. YOU. JOBS. 💰
 
-# TODO: Lastly, we will introduce type conversion. There are functions you can use to
-# TODO: convert things in Python. The first we will use is str(). Try the first
-# TODO: step instructed of using plus signs to concatenate strings to print
-# TODO: `My name is _____ and I am ______ years old.` using the name and age variables.
+# TODO: ######################################################################################
+# TODO: Section 4:
+# TODO: Print `My name is _____ and I am ______ years old.` using the name and age variables.
```

Comment:
Change it to this:

`# TODO: Section 4:
# TODO: Try the first step instructed of using plus signs to concatenate strings to print
# TODO: `My name is _____ and I am ______ years old.` using the name and age variables.
# TODO: However, this time when you're concatenating the age variable, convert it to a
# TODO: string. The print statement will no longer throw an error and the concatenation
# TODO: will work since age was converted to a string.`